www-src/app/replicator/fromDevice/event_synchronizer.coffee

```diff
@@ -0,0 +1,125 @@
+async = require 'async'
+CozyToAndroidEvent = require "../transformer/cozy_to_android_event"
+AndroidCalendarCache = require "../cache/android_calendar_cache"
+AndroidCalendarHandler = require "../../lib/android_calendar_handler"
+log = require('../../lib/persistent_log')
+    prefix: "EventSynchronizer"
+    date: true
+continueOnError = require('../../lib/utils').continueOnError log
+
+module.exports = class EventSynchronizer
+
+    constructor: (@db, @calendarSync) ->
+        @db ?= app.replicator.config.db
+        @calendarSync ?= navigator.calendarsync
+        @androidCalendarHandler = new AndroidCalendarHandler()
+        @cozyToAndroidEvent = new CozyToAndroidEvent()
+        @androidCalendarCache = new AndroidCalendarCache()
+
+    synchronize: ->
+        log.info "synchronize"
+
+        @calendarSync.dirtyEvents @androidCalendarHandler.ACCOUNT, \
+                (err, androidEvents) =>
+            return log.error err if err
+
+            async.eachSeries androidEvents, (androidEvent, cb) =>
+                @change androidEvent, cb
+
+    change: (androidEvent, callback) ->
+        log.info "change"
+
+        if androidEvent.deleted
+            @delete androidEvent, continueOnError callback
+        else
+            # ? androidEvent = ACH.filterOrganizerAttendee androidEvent, @androidCalendarHandler.ACCOUNT
+            @androidCalendarCache.getById androidEvent.calendar_id, \
+                (err, androidCalendar) =>
+                    return log.error err if err
+
+                    if androidEvent._sync_id
+                        @update androidEvent, androidCalendar, continueOnError \
+                                callback
+                    else
+                        @create androidEvent, androidCalendar, continueOnError \
+                                callback
+
+    # Create a new contact in app's pouchDB from newly created phone contact.
+    # @param phoneContact cordova contact format.
+    # @param retry retry lighter update after a failed one.
+    create: (androidEvent, androidCalendar, callback) ->
+        log.info "create"
+
+        cozyEvent = @cozyToAndroidEvent.reverseTransform androidEvent, \
+                androidCalendar
+
+        @db.post cozyEvent, (err, response) =>
+            return callback err if err
+
+#            if err
+#                if err.message is "Some query argument is invalid"
+#                    log.error "While retrying create event in pouch"
+#                    , err
+#                    # Continue with next one.
+#                    return callback null
+#                else
+#                    return callback err
```

Comment:
No commented code in cozy ; )
But I agree with you, I can't remember in which situation this error occurs, and why I bypass it in previous versions. Nevertheless, the upper 'continueOnError' pattern englobe this case, we can safely remove this comments.